shopfloor/tests/test_stock_split.py

```diff
@@ -193,7 +193,7 @@ def _update_qty_in_location(
         # this method adds the quantity to the current quantity, so remove it
         quantity -= sum(quants.mapped("quantity"))
```

Comment:
Don't we need to also add the check `if not quantity: return` here?

Reply:
done